src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.DefaultGlobalUsing.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Two questions:

1. Does it make sense for us to include a license header in files we intend to inject into user's projects? I'm not sure about licensing implications
2. If we should, is this the correct license header?

Reply:
I don't think we should. The generated assembly info files do not, so this should be similar.

Reply:
Do we need an auto-generated header? Or should we not have any headers at all?

Reply:
If there is a header, it should probably tell you how to disable this. Not sure who would actually see this though. You'd have to be looking through the obj folder, right?